mriqc/__init__.py

```diff
@@ -25,7 +25,13 @@
 :abbr:`IQMs (image quality metrics)` to used in :abbr:`QAPs (quality
 assessment protocols)` for :abbr:`MRI (magnetic resonance imaging)`.
 """
-from mriqc._version import __version__
+try:
+    # This option is only available with Python 3.8
+    from importlib.metadata import version as get_version
+except ImportError:
+    from importlib_metadata import version as get_version
+
+__version__ = get_version("mriqc")
```

Comment:
I don't think we want to do this. `_version.py` is still generated by setuptools_scm at build time, and it will definitely be more efficient to read the version from a `.py` file than to query the environment metadata.

Reply:
ok, I reversed that change.
I made it because I ran into an error looking like `mriqc has no module _version`, while running MRIQC through singularity but mounting the repo to test the latest changes, so very specific case.
```
singularity run -e -B $DATADIR:/data:ro \
-B ${OUTDIR}:/out \
-B /oak/stanford/groups/russpold/users/cprovins/code/mriqc/mriqc/:/opt/conda/lib/python3.9/site-packages/mriqc/  
/oak/stanford/groups/russpold/users/cprovins/singularity_images/mriqc-23.1.0.simg /data /out group --nprocs 2 --omp-nthreads 1"

```